TreeTracker/UI/Views/Authentication/SignUp/SignUpViewController.swift

```diff
@@ -33,6 +33,7 @@ class SignUpViewController: UIViewController, KeyboardDismissing, AlertPresentin
             firstNameTextField.delegate = self
             firstNameTextField.keyboardType = .default
             firstNameTextField.returnKeyType = .next
+            firstNameTextField.autocapitalizationType = .words
```

Comment:
Did you try setting the `textContentType` to `.name` for this?

Reply:
It seems that setting the textContentType doesn't autocapitalize on its own - unless on a device with a keyboard it provides a capital letter first. Would you like me to make the changes with both .autocapitalizationType and .textContentType?

Reply:
Ah ok cool - perfect as it is then!